stubs/pika/pika/credentials.pyi

```diff
@@ -1,25 +1,29 @@
 from _typeshed import Incomplete
+from logging import Logger
+from typing import ClassVar
 
-LOGGER: Incomplete
+from .spec import Connection
+
+LOGGER: Logger
 
 class PlainCredentials:
-    TYPE: str
-    username: Incomplete
-    password: Incomplete
-    erase_on_connect: Incomplete
-    def __init__(self, username, password, erase_on_connect: bool = ...) -> None: ...
-    def __eq__(self, other): ...
-    def __ne__(self, other): ...
-    def response_for(self, start): ...
+    TYPE: ClassVar[str]
+    username: str
+    password: str
+    erase_on_connect: bool
+    def __init__(self, username: str, password: str, erase_on_connect: bool = ...) -> None: ...
+    def __eq__(self, other: PlainCredentials) -> bool: ...  # type: ignore[override]
+    def __ne__(self, other: PlainCredentials) -> bool: ...  # type: ignore[override]
```

Comment:
These will work with any object: if compared with an object that isn't an instance of `PlainCredentials`, `__eq__` returns `NotImplemented`, which means that `x == y` (where `x` is an instance of `PlainCredentials` and `y` is "whatever") will evaluate to `False`. So there's no Liskov violation here.

```suggestion
    def __eq__(self, other: object) -> bool: ...
    def __ne__(self, other: object) -> bool: ...
```